gradle.properties

```diff
@@ -14,16 +14,13 @@
 # limitations under the License.
 #
 version=0.11.2
-ideaVersion=2018.1.8
+ideaVersion=PC-2021.2.3
+pythonPlugin=PythonCore:212.5457.59
```

Comment:
I have moved the `pythonPlugin` setting here because it is always required, with or without `pycharmPath`

If it's not set, I get the compile error `package com.jetbrains.python.packaging does not exist`